themes/mlh.zsh-theme

```diff
@@ -47,6 +47,10 @@ if [ -z "$MLH_SHELL_SYMBOL" ]; then
   MLH_SHELL_SYMBOL="$ "
 fi
 
+if [ -z "$MLH_SHELL_SYMBOL_ROOT" ]; then
+  MLH_SHELL_SYMBOL_ROOT="# "
```

Comment:
just to not be breaking, I'll make the default to also be `$`

Reply:
It's not breaking - it's a fix. The Unix convention is to use `#` for superuser prompt. Please reconsider :)

Reply:
it's a breaking fix actually, but let's go for it